starter_code/elevator.js

```diff
@@ -2,18 +2,31 @@ class Elevator {
   constructor(){
     this.floor      = 0;
     this.MAXFLOOR   = 10;
-    this.requests   = [];
+    this.requests   = [Person];
   }
 
-  start() { }
-  stop() { }
+  start() {
+    setInterval(update(), 1000);
+  }
+
+  stop() {clearInterval(start);}
```

Comment:
Not start, you should clear the interval (something that should be declared as this.interval)